JNI_ERR returned from JNI_OnLoad on Android 14

Using version 7849e04, seems to be a new problem on A14

```
 E  FATAL EXCEPTION: main
 E  Process: com.kieronquinn.app.pixelambientmusic, PID: 23686
 E  java.lang.UnsatisfiedLinkError: JNI_ERR returned from JNI_OnLoad in "/data/app/~~3RzKSM6_URpIGvzcY6aWgg==/com.kieronquinn.app.pixelambientmusic-srrS9RwJfu9VYacCWdWCSg==/lib/arm64/libaliuhook.so"
 E      at java.lang.Runtime.loadLibrary0(Runtime.java:1082)
 E      at java.lang.Runtime.loadLibrary0(Runtime.java:1003)
 E      at java.lang.System.loadLibrary(System.java:1661)
 E      at de.robv.android.xposed.XposedBridge.<clinit>(XposedBridge.java:29)
 E      at com.kieronquinn.app.pixelambientmusic.xposed.XposedHooks.hookMethod(XposedHooks.kt:110)
 E      at com.kieronquinn.app.pixelambientmusic.xposed.XposedHooks.setupHooks(XposedHooks.kt:105)
 E      at com.kieronquinn.app.pixelambientmusic.xposed.XposedHooks.init(XposedHooks.kt:69)
 E      at com.kieronquinn.app.pixelambientmusic.xposed.XposedHooks$Companion.setupHooks(XposedHooks.kt:56)
 E      at com.kieronquinn.app.pixelambientmusic.Injector.attachBaseContext(Injector.kt:32)
 E      at android.app.Application.attach(Application.java:346)
 E      at android.app.Instrumentation.newApplication(Instrumentation.java:1282)
 E      at android.app.LoadedApk.makeApplicationInner(LoadedApk.java:1458)
 E      at android.app.LoadedApk.makeApplicationInner(LoadedApk.java:1395)
 E      at android.app.ActivityThread.handleBindApplication(ActivityThread.java:6962)
 E      at android.app.ActivityThread.-$$Nest$mhandleBindApplication(Unknown Source:0)
 E      at android.app.ActivityThread$H.handleMessage(ActivityThread.java:2229)
 E      at android.os.Handler.dispatchMessage(Handler.java:106)
 E      at android.os.Looper.loopOnce(Looper.java:205)
 E      at android.os.Looper.loop(Looper.java:294)
 E      at android.app.ActivityThread.main(ActivityThread.java:8180)
 E      at java.lang.reflect.Method.invoke(Native Method)
 E      at com.android.internal.os.RuntimeInit$MethodAndArgsCaller.run(RuntimeInit.java:552)
 E      at com.android.internal.os.ZygoteInit.main(ZygoteInit.java:946)
```

There's also this one but I think it's Shizuku related not Hook as it's talking about ADB

```
DEBUG  F  #00 pc 0000000000515e68  /apex/com.android.art/lib64/libart.so (art::FaultManager::HandleFault(int, siginfo*, void*)+68) (BuildId: 220b04e747e284569a249db975063ad1)
	 F  #01 pc 00000000000022b0  /apex/com.android.art/lib64/libsigchain.so (art::SignalChain::Handler(int, siginfo*, void*)+548) (BuildId: d2d197c7805a475c4789ddf16c7b263f)
	 F  #02 pc 000000000000088c  [vdso]
	 F  #03 pc 000000000075c3f0  /apex/com.android.art/lib64/libart.so (BuildId: 220b04e747e284569a249db975063ad1)
	 F  #04 pc 000000000059fc24  /apex/com.android.art/lib64/libart.so (bool art::interpreter::DoCall<false>(art::ArtMethod*, art::Thread*, art::ShadowFrame&, art::Instruction const*, unsigned short, bool, art::JValue*)+324) (BuildId: 220b04e747e284569a249db975063ad1)
	 F  #05 pc 00000000004c6710  /apex/com.android.art/lib64/libart.so (void art::interpreter::ExecuteSwitchImplCpp<false>(art::interpreter::SwitchImplContext*)+860) (BuildId: 220b04e747e284569a249db975063ad1)
	 F  #06 pc 0000000000336dd8  /apex/com.android.art/lib64/libart.so (ExecuteSwitchImplAsm+8) (BuildId: 220b04e747e284569a249db975063ad1)
	 F  #07 pc 0000000000151244  /apex/com.android.art/javalib/core-oj.jar (java.nio.HeapByteBuffer.putChar+0)
	 F  #08 pc 0000000000343fa4  /apex/com.android.art/lib64/libart.so (art::interpreter::ExecuteSwitch(art::Thread*, art::CodeItemDataAccessor const&, art::ShadowFrame&, art::JValue, bool) (.__uniq.112435418011751916792819755956732575238)+64) (BuildId: 220b04e747e284569a249db975063ad1)
	 F  #09 pc 0000000000343c48  /apex/com.android.art/lib64/libart.so (art::interpreter::Execute(art::Thread*, art::CodeItemDataAccessor const&, art::ShadowFrame&, art::JValue, bool, bool) (.__uniq.112435418011751916792819755956732575238.llvm.13083381506304087196)+224) (BuildId: 220b04e747e284569a249db975063ad1)
	 F  #10 pc 0000000000343018  /apex/com.android.art/lib64/libart.so (art::interpreter::ArtInterpreterToInterpreterBridge(art::Thread*, art::CodeItemDataAccessor const&, art::ShadowFrame*, art::JValue*)+100) (BuildId: 220b04e747e284569a249db975063ad1)
	 F  #11 pc 000000000059ff10  /apex/com.android.art/lib64/libart.so (bool art::interpreter::DoCall<false>(art::ArtMethod*, art::Thread*, art::ShadowFrame&, art::Instruction const*, unsigned short, bool, art::JValue*)+1072) (BuildId: 220b04e747e284569a249db975063ad1)
	 F  #12 pc 00000000004c6710  /apex/com.android.art/lib64/libart.so (void art::interpreter::ExecuteSwitchImplCpp<false>(art::interpreter::SwitchImplContext*)+860) (BuildId: 220b04e747e284569a249db975063ad1)
	 F  #13 pc 0000000000336dd8  /apex/com.android.art/lib64/libart.so (ExecuteSwitchImplAsm+8) (BuildId: 220b04e747e284569a249db975063ad1)
	 F  #14 pc 00000000003c7050  /system/framework/framework.jar (android.ddm.DdmHandle.putString+0)
	 F  #15 pc 0000000000343fa4  /apex/com.android.art/lib64/libart.so (art::interpreter::ExecuteSwitch(art::Thread*, art::CodeItemDataAccessor const&, art::ShadowFrame&, art::JValue, bool) (.__uniq.112435418011751916792819755956732575238)+64) (BuildId: 220b04e747e284569a249db975063ad1)
	 F  #16 pc 0000000000343c48  /apex/com.android.art/lib64/libart.so (art::interpreter::Execute(art::Thread*, art::CodeItemDataAccessor const&, art::ShadowFrame&, art::JValue, bool, bool) (.__uniq.112435418011751916792819755956732575238.llvm.13083381506304087196)+224) (BuildId: 220b04e747e284569a249db975063ad1)
	 F  #17 pc 0000000000343018  /apex/com.android.art/lib64/libart.so (art::interpreter::ArtInterpreterToInterpreterBridge(art::Thread*, art::CodeItemDataAccessor const&, art::ShadowFrame*, art::JValue*)+100) (BuildId: 220b04e747e284569a249db975063ad1)
	 F  #18 pc 000000000059ff10  /apex/com.android.art/lib64/libart.so (bool art::interpreter::DoCall<false>(art::ArtMethod*, art::Thread*, art::ShadowFrame&, art::Instruction const*, unsigned short, bool, art::JValue*)+1072) (BuildId: 220b04e747e284569a249db975063ad1)
	 F  #19 pc 00000000004c6710  /apex/com.android.art/lib64/libart.so (void art::interpreter::ExecuteSwitchImplCpp<false>(art::interpreter::SwitchImplContext*)+860) (BuildId: 220b04e747e284569a249db975063ad1)
	 F  #20 pc 0000000000336dd8  /apex/com.android.art/lib64/libart.so (ExecuteSwitchImplAsm+8) (BuildId: 220b04e747e284569a249db975063ad1)
	 F  #21 pc 00000000003c5548  /system/framework/framework.jar (android.ddm.DdmHandleHello.handleFEAT+0)
	 F  #22 pc 0000000000343fa4  /apex/com.android.art/lib64/libart.so (art::interpreter::ExecuteSwitch(art::Thread*, art::CodeItemDataAccessor const&, art::ShadowFrame&, art::JValue, bool) (.__uniq.112435418011751916792819755956732575238)+64) (BuildId: 220b04e747e284569a249db975063ad1)
	 F  #23 pc 0000000000343c48  /apex/com.android.art/lib64/libart.so (art::interpreter::Execute(art::Thread*, art::CodeItemDataAccessor const&, art::ShadowFrame&, art::JValue, bool, bool) (.__uniq.112435418011751916792819755956732575238.llvm.13083381506304087196)+224) (BuildId: 220b04e747e284569a249db975063ad1)
	 F  #24 pc 0000000000343018  /apex/com.android.art/lib64/libart.so (art::interpreter::ArtInterpreterToInterpreterBridge(art::Thread*, art::CodeItemDataAccessor const&, art::ShadowFrame*, art::JValue*)+100) (BuildId: 220b04e747e284569a249db975063ad1)
	 F  #25 pc 000000000059ff10  /apex/com.android.art/lib64/libart.so (bool art::interpreter::DoCall<false>(art::ArtMethod*, art::Thread*, art::ShadowFrame&, art::Instruction const*, unsigned short, bool, art::JValue*)+1072) (BuildId: 220b04e747e284569a249db975063ad1)
	 F  #26 pc 00000000004c6710  /apex/com.android.art/lib64/libart.so (void art::interpreter::ExecuteSwitchImplCpp<false>(art::interpreter::SwitchImplContext*)+860) (BuildId: 220b04e747e284569a249db975063ad1)
	 F  #27 pc 0000000000336dd8  /apex/com.android.art/lib64/libart.so (ExecuteSwitchImplAsm+8) (BuildId: 220b04e747e284569a249db975063ad1)
	 F  #28 pc 00000000003c54d4  /system/framework/framework.jar (android.ddm.DdmHandleHello.handleChunk+0)
	 F  #29 pc 0000000000343fa4  /apex/com.android.art/lib64/libart.so (art::interpreter::ExecuteSwitch(art::Thread*, art::CodeItemDataAccessor const&, art::ShadowFrame&, art::JValue, bool) (.__uniq.112435418011751916792819755956732575238)+64) (BuildId: 220b04e747e284569a249db975063ad1)
	 F  #30 pc 0000000000343c48  /apex/com.android.art/lib64/libart.so (art::interpreter::Execute(art::Thread*, art::CodeItemDataAccessor const&, art::ShadowFrame&, art::JValue, bool, bool) (.__uniq.112435418011751916792819755956732575238.llvm.13083381506304087196)+224) (BuildId: 220b04e747e284569a249db975063ad1)
	 F  #31 pc 0000000000343018  /apex/com.android.art/lib64/libart.so (art::interpreter::ArtInterpreterToInterpreterBridge(art::Thread*, art::CodeItemDataAccessor const&, art::ShadowFrame*, art::JValue*)+100) (BuildId: 220b04e747e284569a249db975063ad1)
	 F  #32 pc 000000000059ff10  /apex/com.android.art/lib64/libart.so (bool art::interpreter::DoCall<false>(art::ArtMethod*, art::Thread*, art::ShadowFrame&, art::Instruction const*, unsigned short, bool, art::JValue*)+1072) (BuildId: 220b04e747e284569a249db975063ad1)
	 F  #33 pc 00000000004c6710  /apex/com.android.art/lib64/libart.so (void art::interpreter::ExecuteSwitchImplCpp<false>(art::interpreter::SwitchImplContext*)+860) (BuildId: 220b04e747e284569a249db975063ad1)
	 F  #34 pc 0000000000336dd8  /apex/com.android.art/lib64/libart.so (ExecuteSwitchImplAsm+8) (BuildId: 220b04e747e284569a249db975063ad1)
	 F  #35 pc 000000000003f980  /apex/com.android.art/javalib/core-libart.jar (org.apache.harmony.dalvik.ddmc.DdmServer.dispatch+0)
	 F  #36 pc 0000000000343fa4  /apex/com.android.art/lib64/libart.so (art::interpreter::ExecuteSwitch(art::Thread*, art::CodeItemDataAccessor const&, art::ShadowFrame&, art::JValue, bool) (.__uniq.112435418011751916792819755956732575238)+64) (BuildId: 220b04e747e284569a249db975063ad1)
	 F  #37 pc 0000000000343c48  /apex/com.android.art/lib64/libart.so (art::interpreter::Execute(art::Thread*, art::CodeItemDataAccessor const&, art::ShadowFrame&, art::JValue, bool, bool) (.__uniq.112435418011751916792819755956732575238.llvm.13083381506304087196)+224) (BuildId: 220b04e747e284569a249db975063ad1)
	 F  #38 pc 0000000000319954  /apex/com.android.art/lib64/libart.so (artQuickToInterpreterBridge+960) (BuildId: 220b04e747e284569a249db975063ad1)
	 F  #39 pc 0000000000334788  /apex/com.android.art/lib64/libart.so (art_quick_to_interpreter_bridge+88) (BuildId: 220b04e747e284569a249db975063ad1)
	 F  #40 pc 000000000031de80  /apex/com.android.art/lib64/libart.so (art_quick_invoke_static_stub+640) (BuildId: 220b04e747e284569a249db975063ad1)
	 F  #41 pc 00000000002a1294  /apex/com.android.art/lib64/libart.so (art::ArtMethod::Invoke(art::Thread*, unsigned int*, unsigned int, art::JValue*, char const*)+220) (BuildId: 220b04e747e284569a249db975063ad1)
	 F  #42 pc 0000000000505d4c  /apex/com.android.art/lib64/libart.so (art::detail::ShortyTraits<(char)76>::Type art::ArtMethod::InvokeStatic<(char)76, (char)73, (char)76, (char)73, (char)73>(art::Thread*, art::detail::ShortyTraits<(char)73>::Type, art::detail::ShortyTraits<(char)76>::Type, art::detail::ShortyTraits<(char)73>::Type, art::detail::ShortyTraits<(char)73>::Type)+84) (BuildId: 220b04e747e284569a249db975063ad1)
	 F  #43 pc 000000000050539c  /apex/com.android.art/lib64/libart.so (art::Dbg::DdmHandleChunk(_JNIEnv*, unsigned int, art::ArrayRef<signed char const> const&, unsigned int*, std::__1::vector<unsigned char, std::__1::allocator<unsigned char> >*)+272) (BuildId: 220b04e747e284569a249db975063ad1)
	 F  #44 pc 000000000000ba9c  /apex/com.android.art/lib64/libadbconnection.so (adbconnection::AdbConnectionState::HandleDataWithoutAgent(art::Thread*)+1024) (BuildId: 1147b1f7bff3ebcbe6c22cc4acaa76cb)
	 F  #45 pc 000000000000aca8  /apex/com.android.art/lib64/libadbconnection.so (adbconnection::AdbConnectionState::RunPollLoop(art::Thread*)+1868) (BuildId: 1147b1f7bff3ebcbe6c22cc4acaa76cb)
	 F  #46 pc 0000000000008f7c  /apex/com.android.art/lib64/libadbconnection.so (adbconnection::CallbackFunction(void*)+1496) (BuildId: 1147b1f7bff3ebcbe6c22cc4acaa76cb)
	 F  #47 pc 00000000000ca25c  /apex/com.android.runtime/lib64/bionic/libc.so (__pthread_start(void*)+204) (BuildId: 3d8ab34529f579f97f90b4d6e80c259f)
	 F  #48 pc 000000000005cbc0  /apex/com.android.runtime/lib64/bionic/libc.so (__start_thread+64) (BuildId: 3d8ab34529f579f97f90b4d6e80c259f)
```

--- a/core/build.gradle.kts
+++ b/core/build.gradle.kts
@@ -20,7 +20,7 @@ plugins {
 }
 
 dependencies {
-    implementation("org.lsposed.lsplant:lsplant:4.0-aliucord.1")
+    implementation("org.lsposed.lsplant:lsplant:5.2-aliucord.1")
     implementation("io.github.vvb2060.ndk:dobby:1.2")
 
     androidTestImplementation("androidx.test.ext:junit:1.1.3")

--- a/settings.gradle.kts
+++ b/settings.gradle.kts
@@ -5,8 +5,8 @@ pluginManagement {
         mavenCentral()
     }
     plugins {
-        id("com.android.application") version "7.2.2"
-        id("com.android.library") version "7.2.2"
+        id("com.android.application") version "7.3.1"
+        id("com.android.library") version "7.3.1"
     }
 }
 dependencyResolutionManagement {

